cli/src/commands/init.js

```diff
@@ -242,6 +280,12 @@ const command = {
             type: 'boolean',
             default: false,
         },
+        typeScript: {
```

Comment:
_Suuuper_ subjective and equally minor since the alias is set up, and maybe I’m crazy, but I’ll share in case it’s in the back of your mind too: `—typeScript` doesn’t feel right to me as a flag or variable name 😅 I think of it as `typescript` or `ts` — it seems like you used `—typescript` too when you wrote out CLI command script for your workflow